deploy/charts/openebs-monitoring/README.md

```diff
@@ -47,9 +45,9 @@ _See [helm install](https://helm.sh/docs/helm/helm_install/) for command documen
 
 By default this chart installs additional, dependent charts:
 
-| Repository | Name | Version |
-|------------|------|---------|
-| https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack | kube-prometheus-stack | 15.4.* |
+| Repository                                                                                 | Name                  | Version |
+| ------------------------------------------------------------------------------------------ | --------------------- | ------- |
+| https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack | kube-prometheus-stack | 15.4.\* |
```

Comment:
Is the version written as `15.4.\*` correct?

Reply:
Oh, I think it is a keyboard error 😛 . WIll fix it.

Reply:
Checked it. While formatting in vs-code,  `\` was being added before `*`. So made the changes to `15.4.*` .